test/acceptance/tests/basic_test.go

```diff
@@ -78,11 +85,15 @@ func TestBasic(t *testing.T) {
 
 			terraform.InitAndApply(t, setupTerraformOptions)
 
+			var setupCfg SetupConfig
+			require.NoError(t, UnmarshalTF("./setup", &setupCfg))
+
 			clientServiceName := fmt.Sprintf("test_client_%s", suffix)
 			preexistingLambdaServiceName := fmt.Sprintf("preexisting_%s", setupSuffix)
-			lambdaServiceName := fmt.Sprintf("example_%s", suffix)
-			prodLambdaServiceName := fmt.Sprintf("example_%s-prod", suffix)
-			devLambdaServiceName := fmt.Sprintf("example_%s-dev", suffix)
+			mtlServiceName := fmt.Sprintf("mtl_example_%s", suffix)
+			prodLambdaServiceName := fmt.Sprintf("mtl_example_%s-prod", suffix)
+			devLambdaServiceName := fmt.Sprintf("mtl_example_%s-dev", suffix)
+			ltmServiceName := fmt.Sprintf("ltm_example_%s", suffix)
```

Comment:
Could you either expand the `ltm` and `mtl` names or add a comment with their meaning?